dptb/nnops/ema.py

```
@@ -0,0 +1,82 @@
+            # warmup: effective decay ramps up so early steps are not dominated
+            # by the random initialization
+            decay = min(decay, (1 + self.num_updates) / (10 + self.num_updates))
+        one_minus_decay = 1.0 - decay
+        with torch.no_grad():
+            for s, p in zip(self.shadow, self._params):
+                s.add_(one_minus_decay * (p - s))
+
+    def copy_to(self, parameters=None):
+        parameters = self._params if parameters is None else [p for p in parameters if p.requires_grad]
+        with torch.no_grad():
+            for s, p in zip(self.shadow, parameters):
```

Ruff (0.15.21)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)